core/server/apps/private-blogging/lib/middleware.js

```diff
@@ -70,7 +70,7 @@ privateBlogging = {
         // NOTE: Redirect to /private if the session does not exist.
         privateBlogging.authenticatePrivateSession(req, res, function onSessionVerified() {
             // CASE: RSS is disabled for private blogging e.g. they create overhead
-            if (req.path.lastIndexOf('/rss/', 0) === 0 || req.path.lastIndexOf('/rss/') === req.url.length - 5) {
+            if (req.path.match(/\/rss\/?$/)) {
```

Comment:
I just remembered that rss is paginated e.g. `/rss/2/`.
Regex has to be extended for that. Sorry, forgot about it 🙈

Reply:
It's fine, I'll fix it :)